pkg/image/registry/image/strategy.go

```diff
@@ -49,6 +52,8 @@ func (s imageStrategy) PrepareForCreate(ctx apirequest.Context, obj runtime.Obje
 		newImage.DockerImageManifest = ""
 		newImage.DockerImageConfig = ""
 	}
+
+	removeManagedSignatureAnnotation(newImage)
```

Comment:
since it's now only going to drop the one annotation, i think i'm ok w/ doing this on create, for the (unlikely) backup/restore use case.  @deads2k ?

Reply:
> since it's now only going to drop the one annotation, i think i'm ok w/ doing this on create, for the (unlikely) backup/restore use case. @deads2k ?

Ok.